tools/run_tests/run_interop_tests.py

```diff
@@ -733,6 +738,63 @@ def aggregate_http2_results(stdout):
     'percent': 1.0 * passed / (passed + failed)
   }
 
+def push_to_gke_registry(image, gcr_path, gcr_tag='latest', with_files=[]):
```

Comment:
Again, our testing scripts are already way too complicated and they are hard enough to maintain already. I understand the intention here (and it is reasonable), but in general, I'd like to set a really high bar for adding functionality to run_*_tests.py scripts.
I'd prefer to have some very minimal changes to run_interop_tests.py to help you obtain the information you need (e.g. get the list of commands you need, list of identifiers of the docker images and a flag for avoiding deleting of them) and all other logic would be accomplished by a separate (and specialized) script.